python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -218,3 +222,16 @@ def _from_points_xy(cls, points_xy: ColumnBase):
             ),
             meta,
         )
+
+    @cached_property
+    def memory_usage(self) -> int:
```

Comment:
This is from another [PR](https://github.com/rapidsai/cuspatial/pull/604). Should remove.